BMP-masks/masks.py

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/python3
+"""This module provides two functions, either to create mask based on provided bit offset and bits per color number or
+extract these values from provided mask.
+"""
+
+
+def get_mask(bit_offset: int, bits_per_color: int = 8) -> int:
+    """Calculate the mask for a specific color and bit offset.
+    Args:
+        bit_offset: the number of bit offset for specific color value.
+        bits_per_color: the number of bits to encode color value.
+    Returns:
+        The string which represents bit mask for specific color.
+    Raises:
+        TypeError: if bit_offset or bits_per_color are not a number.
+        ValueError: if bit_offset or bits_per_color are negative.
+    """
+    if not (isinstance(bit_offset, int) and isinstance(bits_per_color, int)):
+        raise TypeError
+    if bit_offset < 0 or bits_per_color <= 0:
+        raise ValueError
+    val = (1 << bits_per_color) - 1
+    return val << (4 * bit_offset)
```

Comment:
Nie ma potrzeby mnożyć * 4. Jak nazwa zmiennej wskazuje, to `bit_offset`, nie `nibble_offset`.